app.R

```diff
@@ -0,0 +1,843 @@
+                    "United Kingdom-Females" = "UK-Females",
+                    "United Kingdom-Males" = "UK-Males",
+                    "United States-Females" = "US-Females",
+                    "United States-Males" = "US-Males"
+                  )
+                )
+              )
+            ),
+            column(
+              width = 8,
+              offset = 0,
+              plotOutput("survivalChart1")
+            )
+          )
+        ),
+        #### Static Pyramid Page----
+        tabItem(
+          tabName = "staticPry",
+          h2("Static Population Pyramids"),
+          p("Population pyramids are common visualizations for life data. They
+            are composed of two histograms, breaking a countries population into
+            two sub-collections, typically by sex."),
+          tabsetPanel(
+            id = "cohortPyramid",
+            type = "tabs",
+            ##### Country to Country ----
+            tabPanel(
+              title = "Country to Country",
+              br(),
+              p("Compare each country's population pryamid to the other countries.
+                What do you notice?"),
+              fluidRow(
+                column(
+                  width = 4,
+                  offset = 0,
+                  wellPanel(
+                    radioButtons(
+                      inputId = "upperCountry",
+                      label = "Select the country for the upper graph",
+                      choices = c(
+                        "China",
+                        "United Kingdom",
+                        "United States"
+                      ),
+                      selected = "China"
+                    ),
+                    radioButtons(
+                      inputId = "lowerCountry",
+                      label = "Select the country for the lower graph",
+                      choices = c(
+                        "China",
+                        "United Kingdom",
+                        "United States"
+                      ),
+                      selected = "United Kingdom"
+                    )
+                  )
+                ),
+                column(
+                  width = 8,
+                  offset = 0,
+                  plotOutput("upperPyramid"),
+                  plotOutput("lowerPyramid")
+                )
+              )
+            ),
+            ##### Country & Sex ----
+            tabPanel(
+              title = "Country & Sex Comparisons",
+              br(),
+              p("Population pryamids can also be mixed and matched across countries
+                and sex. Create various pryamids to compare the different pairings
+                of country and sex. What do you notice?"),
+              fluidRow(
+                column(
+                  width = 4,
+                  offset = 0,
+                  wellPanel(
+                    p("Select which Country-Sex pairings you want to compare."),
```

Comment:
Does the style guide allow you to bold likes 219, 222, and 235? These labels kind of blend into the rest of the text and I think it would be nice to have them stand out in some way since they are not part of the list of things to compare.

Reply:
Generally, you want to try to limit usages of bold (`strong`) and italics (`em`) so that they retain their special case status. Over using them can dilute their meaning. If we were to be extremely judicious with our usage we might be able to do something, but we would need something specific. Making a whole label bold seems like a step too far. Perhaps there's something else we can do to try to resolve the issue you're describing.